templates/invoices/new.html

```diff
@@ -378,6 +378,15 @@ <h2>Fakturadelar</h2>
         }
     });
 
+    function validDate() {
+        const date = document.getElementById("invoice-date");
+        const dueDate = document.getElementById("invoice-due-date");
+        if (dueDate.value < date.value)
+            dueDate.setCustomValidity("Kan inte vara innan fakturadatum.");
+        else
+            dueDate.setCustomValidity("");
```

Comment:
```suggestion
        if (dueDate.value < date.value) {
            dueDate.setCustomValidity("Kan inte vara innan fakturadatum.");
        } else {
            dueDate.setCustomValidity("");
        }
```
better for future extensibility

Reply:
I don't agree 😠